cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/power/HostActivationExample.java

```diff
@@ -137,7 +137,8 @@ private DatacenterBroker createBroker() {
 
         /*Indicates that idle VMs must be destroyed right away (0 delay).
         * This forces the Host to become idle*/
-        broker.setVmDestructionDelay(0.0);
+        //broker.setVmDestructionDelay(0.0);
```

Comment:
You don't need to comment removed/changed lines.
Just do it and I make sure to review.